plugin/fulltext/mysql-test/ngram/ngram_1.test

```diff
@@ -0,0 +1,64 @@
+# Test fulltext with ngram parser
+
+--source include/no_valgrind_without_big.inc
+--source include/have_innodb_max_16k.inc
+
+
+# Because we have inserted row size > 2k, we require page size > 4k.
+if (`SELECT COUNT(*) = 1 FROM performance_schema.global_status WHERE LOWER(variable_name) = 'innodb_page_size' AND variable_value = 4096`)
+{
+  --skip Test requires InnoDB with Page size > 4k
+}
+
+LET $per_table = `select @@innodb_file_per_table`;
+
+--echo # Test Case : With default row type
+LET $table_row_format =  ROW_FORMAT=DEFAULT ;
+--source ./include/ngram.inc
```

Comment:
I think that we should use `ngram.inc` directly as the test case and remove this ‘wrapper’.
```sql
--source include/innodb_row_format.inc
```
would cover all `ROW_FORMAT` except `ROW_FORMAT=COMPRESSED`, which I do not think is useful.
I do not think that we need either of these:
```sql
--source include/no_valgrind_without_big.inc
--source include/have_innodb_max_16k.inc
```
If we wanted to cover all `innodb_page_size` in this test (I do not see any need for it) then we could use one of the following:
```sql
--source include/innodb_page_size_small.inc
--source include/innodb_page_size.inc
```
The 32k and 64k page sizes are not compatible with `ROW_FORMAT=COMPRESSED`.

Reply:
Previously I took most of the test from MySQL code, but while I tried to address review comments, I realized that those tests are too inflated. They try to test too much. There were even tests for index renaming which had absolutely nothing to do with ngram plugin. There were many repeated tests which tested the same thing. So I decided to rewrite them from scratch.

There is no need to check various row formats and page sizes in tests for a parser plugin, so I didn't add them.